lib/biomarker.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Use the same title for biomarker search page and biomarker object page for that biomarker
+ */
```

Comment:
We’ve (“I’ve”?) been using JSDoc for normal function comments (as opposed to React component comments which have props documented in propTypes). That way when someone else using VSCode hovers over the function name where it gets called, they can see the parameters and return values documented in the popover without having to go to the function definition itself.